core/gen.go

```diff
@@ -0,0 +1,32 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
```

Comment:
This is all in #842 - i assume you'll rebase and remove this when #842 is merge?

Reply:
Yes. This is merged now.